_posts/2017-03-28-CocoaDocs-Documentation-Sunsetting.markdown

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  "CocoaDocs documentation sun-setting"
+author: orta
+categories: cocoadocs documentation cocoapods.org
+---
+
+TL;DR: _CocoaDocs_ will stop generating DocSets for new libraries and versions using CocoaPods towards the end of May.
+
+I started [CocoaDocs][cd] [back in][cd_tweet] February 2013, and have been running and maintaining it for the last 4 years. CocoaDocs started as a way to generate documentation for Objective-C projects via [Appledoc][appledoc]. When Swift came out, [Jazzy][jazzy] support was added to the app. Since then, CocoaDocs has grown organically over the years into a tool to [generate useful metadata][cd_post] for any CocoaPods library used in the CocoaPods.org website.
+
+I intend to remove the support for generating the HTML pages, and the DocSet generation for new libraries uploaded to trunk as by the end of May.
```

Comment:
Few typos/tweaks:
- /generating the HTML pages, and the DocSet generation/generating HTML pages and DocSets/
- /trunk/Trunk/
- /as by/by